app/src/main/java/framgia/com/myeditor/screen/home/HomeFragment.java

```diff
@@ -0,0 +1,82 @@
+package framgia.com.myeditor.screen.home;
+
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import framgia.com.myeditor.databinding.FragmentHomeBinding;
+import framgia.com.myeditor.utils.rx.SchedulerProvider;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class HomeFragment extends Fragment {
+
+    private HomeViewModel mViewModel;
+    private FragmentHomeBinding mBinding;
+
+    public HomeFragment() {
+        // Required empty public constructor
+    }
+
+    public static HomeFragment newInstance() {
+        return new HomeFragment();
+    }
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
+            Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        mBinding = DataBindingUtil.inflate(inflater, R.layout.fragment_home, container, false);
+        initBinding();
+        return mBinding.getRoot();
+    }
+
+    private void initBinding() {
+        mViewModel = new HomeViewModel(getContext(), mBinding.viewPagerImage,
+                new ImageRepository(ImageRemoteDataSource.getsInstance(),
+                        ImageLocalDataSource.getsInstance()));
+        mViewModel.setSchedulerProvider(SchedulerProvider.getInstance());
+        mBinding.setViewModel(mViewModel);
+        initViews();
+    }
+
+    private void initViews() {
+        mBinding.recyclerCollection.setLayoutManager(
+                new LinearLayoutManager(getContext().getApplicationContext(),
+                        LinearLayoutManager.HORIZONTAL, false));
```

Comment:
e nên set thẳng layoutManager vào xml cho gọn code
https://stackoverflow.com/questions/35679776/how-to-set-recyclerview-applayoutmanager-from-xml